spec/lib/traffic_source_parser/parser/utmz_parser_spec.rb

```diff
@@ -18,15 +18,45 @@
         it_behaves_like 'a traffic source campaign parser'
       end
 
-      context 'when value is from social' do
+      context 'when medium is display' do
+        let(:cookie) { ' 186095035.1415927951.1.1.utmcsr=google|utmgclid=CI_WsND2-MECFezm7Aod6DsASw|utmccn=scm_beta|utmcmd=display|utmcct=Display_integra' }
+        let(:campaign) { 'scm_beta' }
+        let(:medium) { 'display' }
+        let(:channel) { 'Display' }
+        let(:source) { 'google' }
+
+        it_behaves_like 'a traffic source campaign parser'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.